### ISSUE TITLE
Support for speaker-view picture-in-picture on chrome

### ISSUE THREAD

**Issue by cyanomys** [jitsi/jitsi-meet#6534]
Chrome and chromium-based browsers have an awesome picture-in-picture feature which can be enabled by default, via flags, or via an official browser extension, depending on which browser you are using.  The feature can be used to pop out any video into a small overlay window that you can view while browsing other tabs. It works on Jitsi, but it only pops out one user's video, no matter who is speaking.  It would be ideal if Jitsi could be adapted to send the speaker's video to this feature, so that when someone starts speaking the popout would show the video of whichever person is speaking instead of just one user all the time.  This would be a similar to Zoom or Skype's picture-in-picture features, and would make Jitsi much more useful for people who use it to conference while playing games, working on documents, etc.

**Comment by damencho** [comment]
Isn't it working if you pop out the large video?

**Comment by cyanomys** [comment]
No, unfortunately it only seems to latch on to the video that was currently showing and does not change with the speaker as the large video normally does.
